src/commands/bootstrap.ts

```diff
@@ -53,6 +53,33 @@ export function bootstrap({
         console.error(error);
       }
     }
+
+    if (reaction.emoji.name === '📌') {
+      try {
+        reaction.users.fetch().then((users) => {
```

Comment:
Should be able to do this as an `await`, rather than having to use `.then`.

Reply:
Have updated `.then` to `await`.